test-infra/camel-test-infra-ftp/src/main/java/org/apache/camel/test/infra/ftp/services/embedded/FtpsEmbeddedInfraService.java

```diff
@@ -19,13 +19,21 @@
 
 import java.io.File;
 
+import org.apache.camel.spi.annotations.InfraService;
+import org.apache.camel.test.infra.ftp.services.FtpInfraService;
 import org.apache.ftpserver.FtpServerFactory;
 import org.apache.ftpserver.listener.ListenerFactory;
 import org.apache.ftpserver.ssl.SslConfigurationFactory;
 
+@InfraService(service = FtpInfraService.class, serviceAlias = { "ftps" })
 public class FtpsEmbeddedInfraService extends FtpEmbeddedInfraService {
```

Comment:
This is probably the only one I am a bit concerned about: by using an embedded service, we'll end up bringing all the dependencies used for that (this, for instance may expose us to a higher incidence of CVEs). 

I don't think we must solve this now, but it's something I think we should consider for the future versions.

Reply:
good point, same for Artemis, but I do not think we have many embedded implementations, most probably it can be refactored in the future to use testcontainers